src/main/java/com/teamcautionrobotics/robot/Cargo.java

```diff
@@ -0,0 +1,55 @@
+
+package com.teamcautionrobotics.robot;
+
+import edu.wpi.first.wpilibj.*;
+
+public class Cargo {
+
+    public enum FunnelRollerSetting {
+        THROUGH(1), BACK(-1), STOP(0);
+
+        public final double power;
+
+        private FunnelRollerSetting(double power) {
+            this.power = power;
+        }
+    }
+
+    // motor object
+    private final VictorSP funnelRoller;
+    private final VictorSP conveyorRoller;
+    // pneumatics object
+    private final Solenoid deployFunnelRoller;
+    private final Solenoid deployExitFlap;
```

Comment:
*thinks back to questions about Java identifier rules (S. Tierney, personal communication, 13 February, 2019).*

These use of "deploy" in these names may not be the clearest. "deploy"'s verbness suggests that these might be a method or a flag to do something. Instead consider `funnelRollerPiston` and `exitFlapPiston`.